packages/builder-util-runtime/src/publishOptions.ts

```diff
@@ -146,6 +146,39 @@ export interface GenericServerOptions extends PublishConfiguration {
   readonly useMultipleRangeRequest?: boolean
 }
 
+/**
+ * Keygen options.
+ * https://keygen.sh/
+ * Define `KEYGEN_TOKEN` environment variable.
+ */
+export interface KeygenOptions extends PublishConfiguration {
+  /**
+   * The provider. Must be `keygen`.
+   */
+  readonly provider: "keygen"
+
+  /**
+   * Keygen account's UUID
+   */
+  readonly account: string
+
+  /**
+   * Keygen product's UUID
+   */
+  readonly product: string
+
+  /**
+   * The channel.
+   * @default stable
+   */
+  readonly channel?: string | null
```

Comment:
Should this match what the API allows?

```ts
readonly channel?: 'stable' | 'rc' | 'beta' | 'alpha' | 'dev' | null
```

Fine either way. The API will reject channels that do not match one of the above.

Reply:
Great point! I wanted to leave it dynamic so that any updates from your side wouldn't necessarily require updating electron-builder as well.
That being said, it's much more handy/efficient to have the channel info all provided up front. Will update.